bin/test.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env bash
+
+#------------------------------------------------------------------------------
+# Script
+#------------------------------------------------------------------------------
+
+SCRIPT_DIR=$(dirname $0)
+
+print_help () {
+  echo "Usage: $0 "
+  echo "       [--targets <"target1 target2 ..." | help>]"
+  echo "       [--slow]"
+  echo "       [--quick]"
+  echo "       [--cabal-build-options <option>]"
+  echo "       -- <hspec options or test names>"
+  echo
+  echo "--targets: targets to run, use 'help' for list of targets"
+  echo "--slow: take longer but run more tests"
+  echo "--quick: faster  but run fewer tests"
```

Comment:
is it fewer test suites or fewer number of quickcheck iterations ?

Reply:
It is a to-do, does not anything right now. Just retained this from benchmarks script.